src/src/com/microsoft/aad/adal/SerializedTokenCacheItem.java

```diff
@@ -0,0 +1,40 @@
+package com.microsoft.aad.adal;
+
+import com.google.gson.annotations.SerializedName;
+
+final class SerializedTokenCacheItem {
```

Comment:
You could just register custom serializer into GSON from the TokenCacheItem, see `registerTypeAdapter`, it would be much cleaner